highlighters/main/main-highlighter.zsh

```diff
@@ -455,7 +468,7 @@ _zsh_highlight_main_highlighter_highlight_list()
       fi
     fi
 
-    if true; then
+    if (( in_alias == 0 )); then
```

Comment:
Does this DTRT on `alias foo='x && y' y=ls` followed by `% foo`?

Reply:
I think more interesting would be `y=sudo` and yes.